.goreleaser.yml

```diff
@@ -1,33 +1,61 @@
 # You can find the GoReleaser documentation at http://goreleaser.com
 project_name: exercism
 
+env:
+  - CGO_ENABLED=0
 builds:
-- env:
-    - CGO_ENABLED=0
-  main: ./exercism/main.go
-  goos:
-    - darwin
-    - linux
-    - windows
-    - freebsd
-    - openbsd
-  goarch:
-    - amd64
-    - 386
-    - arm
-    - arm64
-    - ppc64
-  goarm:
-    - 5
-    - 6
-  ignore:
-    - goos: openbsd
-      goarch: arm
-    - goos: freebsd
-      goarch: arm
+  - id: release-build
+    main: ./exercism/main.go
+    mod_timestamp: '{{ .CommitTimestamp  }}'
+    flags:
+      - -trimpath # removes file system paths from compiled executable
+      - -buildvcs=false # omit vcs information from binaries
+    ldflags:
+      - '-s -w -X {{ .ModulePath }}/cmd.Version={{.Version}}' # sets binary version to match tag without 'v' prefix
```

Comment:
This is to replace the version with the tag, right? I'm not opposed to this change, but I also think there is some value in having the version directly defined in the code. With the current setup, there could be a disconnect between the tag version and the version in the file, right?

Reply:
> This is to replace the version with the tag, right? I'm not opposed to this change, but I also think there is some value in having the version directly defined in the code. With the current setup, there could be a disconnect between the tag version and the version in the file, right?

Yes, the version is injected at build time. I agree that the version should be properly set within `cmd/version.go` before triggering the release. Seeing as bumping the version and CHANGELOG.md is part of the release process I figured the person releasing would bump the version before triggering the release. 

That said, this does add a little disconnect between the tag version and the version file if the release workflow is not followed directly. Which can also be a crutch if the tag version is correct but the version.go file is not.

If you this adds possible confusion or room for error we can drop the change. I thought locally (manual release) would be an issue but it looks like the process is to create the tag  before running goreleaser, which will result in the correct version being injected locally as well.

```
# Create a new tag on the main branch and push it
git tag -a "${TAG_NAME}" -m "Trying out GoReleaser"
git push origin "${TAG_NAME}"

# Build and release
goreleaser --clean
```

Reply:
> If you this adds possible confusion or room for error we can drop the change.

I think I would prefer that. We can always add it later. Ideally we'd be able to automate as much of the version bumping too at some point.

Reply:
Yeah good call. The change has been removed. I rebased and squashes commits a bit but you should be able to see the last commit where I revert